x/dualstaking/keeper/delegate.go

```diff
@@ -382,10 +376,12 @@ func (k Keeper) unbond(ctx sdk.Context, delegator, provider, chainID string, amo
 		}
 	}
 
-	if err := validateCoins(amount); err != nil {
-		return err
-	} else if amount.IsZero() {
-		return nil
+	if err := utils.ValidateCoins(ctx, k.stakingKeeper.BondDenom(ctx), amount, false); err != nil {
+		return utils.LavaFormatWarning("failed to unbond: coin validation failed", err,
+			utils.Attribute{Key: "delegator", Value: delegator},
+			utils.Attribute{Key: "provider", Value: provider},
+			utils.Attribute{Key: "amount", Value: amount.String()},
```

Comment:
same

Reply:
Fixed in 4a9bd33080264c1f8f20a13cfead74ce23227bca